### DISCUSSION TITLE
Connection Problem

### DISCUSSION THREAD

**Discussion by tihamc** [pauldex/sqlalchemy-firebird#66] [Q&A]
Hi,

I'm trying to use sqlalchemy-firebird, but i'm getting this error: sqlalchemy.exc.DatabaseError: (firebird.driver.types.DatabaseError) Unable to complete network request to host "192.168.1.28".

DB server
Windows server
Firebird 3.0.7
Port 3050
ip: 192.168.1.28
Database path: E:/Work/Banco/SGHDADOS.639

APP Server
Centos 7
Python 3.10.12
String conection: firebird+firebird://sysdba:{password}@192.168.1.28/E:/Work/Banco/SGHDADOS.639
Driver libfbclient.so.2
Requirements.txt
alembic==1.10.2
blinker==1.5
click==8.1.3
colorama==0.4.6
dnspython==2.3.0
email-validator==2.0.0.post2
fdb==2.0.2
firebird-base==1.6.1
firebird-driver==1.10.0
Flask==2.2.3
Flask-Login==0.6.2
Flask-Mail==0.9.1
Flask-Migrate==4.0.4
Flask-Moment==1.0.5
Flask-SQLAlchemy==3.0.3
Flask-WTF==1.1.1
future==0.18.3
greenlet==2.0.2
idna==3.4
importlib-metadata==6.0.0
importlib-resources==5.12.0
itsdangerous==2.1.2
Jinja2==3.1.2
Mako==1.2.4
MarkupSafe==2.1.2
packaging==23.0
protobuf==4.21.12
psycopg2-binary==2.9.9
python-dateutil==2.8.2
python-dotenv==0.21.1
six==1.16.0
SQLAlchemy==2.0.5.post1
sqlalchemy-firebird==2.1
typing_extensions==4.5.0
Werkzeug==2.2.3
WTForms==3.0.1
zipp==3.15.0

**Comment by pauldex** [comment]
Did you get this resolved?
